src/main/java/by/itech/lab/supplier/domain/Address.java

```diff
@@ -2,40 +2,42 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import lombok.AllArgsConstructor;
-import lombok.Getter;
+import lombok.Builder;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
 
-import javax.persistence.*;
-import java.util.HashSet;
-import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
 
+
+@Data
 @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
```

Comment:
It seems like this annotation should not be used here. It's dao level, not the controller.

Reply:
+++